content/03-how-tos/08-how-check-hash-windows.mdx

```diff
@@ -0,0 +1,31 @@
+---
+title: How to verify a checksum on Windows 10
+metaTitle: How to verify a checksum on Windows 10
+---
+
+## Overview
+
+This page shows how to make sure that the installer has not been corrupted during the download process. Microsoft Excel is a useful comparison tool.
+
+## Get the hash from the website
+
+The image shows the wallet page; the same principles apply to the client page.
+![Windows installer](../images/9-wallet-checksum.png)
+
+The SHA-256 value is below the installer.
+Open a blank spreadsheet and copy and paste the hash from the website to cell A1 of the spreadsheet.
```

Comment:
Another convenient way would be to paste it into a text editor (eg. notepad) and use the "Find" feature to check for a match. (And it works is command line terminals as well on every platform.)